Gemfile.lock

```diff
@@ -36,14 +36,15 @@ GIT
 
 GIT
   remote: https://github.com/18F/saml_idp.git
-  revision: bdf8e1f93707e413ecbd0f48d803e18812e19f90
-  tag: 0.23.5-18f
+  revision: 32e9be98c30bc5d01b4088500e4d518f724aadc5
+  tag: 0.23.6-18f
   specs:
-    saml_idp (0.23.5.pre.18f)
+    saml_idp (0.23.6.pre.18f)
       activesupport
       builder
       faraday
       nokogiri (>= 1.10.2)
+      ostruct
```

Comment:
so this wasn't a new add overall for the `ostruct` gem, we already had it, so I'm wondering if the warning was already taken care of? Or this won't affect it?

Reply:
hm ok well we added as an explicit dependecy in https://github.com/18F/identity-idp/pull/11605/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR472, but hopefully this helps squash the warning!

Reply:
Is it due to `ostruct` previously only being a dependency of test/dev dependencies?